src/openrct2/peep/Peep.h

```diff
@@ -38,6 +38,8 @@
 #define PEEP_MAX_ENERGY 128
 #define PEEP_MAX_ENERGY_TARGET 255 // Oddly, this differs from max energy!
 
+struct rct_tile_element;
```

Comment:
I think it'll be nicer to have forward declarations directly after the include list.